service/helpers.js

```diff
@@ -14,6 +14,14 @@ const formatDateWithSign = (date, sign = '/') => {
   return `${`00${day}`.slice(-2)}${sign}${`00${month}`.slice(-2)}${sign}${`00${year}`.slice(-2)}`;
 };
 
+const compareDescendingDMY = (startDate, endDate) => {
+  const resetStartDate = new Date(startDate);
```

Comment:
Consider using this, since we already have it in the project - https://date-fns.org/v2.25.0/docs/compareDesc